src/app/features/new-hire/personal-info/personal-info.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { Location } from '@angular/common';
+
+import { PersonalInfoComponent } from './personal-info.component';
+
+let mockLocation: any;
+
+describe('PersonalInfoComponent', () => {
+  let component: PersonalInfoComponent;
+  let fixture: ComponentFixture<PersonalInfoComponent>;
+
+  beforeEach(async(() => {
+    mockLocation = {
+      back: jasmine.createSpy('back'),
+    };
+
+    TestBed.configureTestingModule({
+      declarations: [PersonalInfoComponent],
+      providers: [
+        { provide: Location, useValue: mockLocation }
+      ],
+      schemas: [NO_ERRORS_SCHEMA],
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PersonalInfoComponent);
+
+    component = fixture.componentInstance;
+  });
+
+  it('should create', () => {
+    expect(component)
+      .toBeTruthy();
+  });
+
+  describe('logic', () => {
```

Comment:
you do not need to create an additional describe layer. Moreover this adds an additional information to the error message that can mislead : 
'PersonalInfoComponent logic back expected ... but actual was ...'

Reply:
done